src/xcode/ENA/ENA/Source/Services/HealthCertificateValidation/BoosterNotificationsService.swift

```diff
@@ -0,0 +1,37 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+import class CertLogic.Rule
+
+protocol BoosterNotificationsServiceProviding {
+	func downloadBoosterNotificationRules(
+		completion: @escaping (Result<[Rule], HealthCertificateValidationError>) -> Void
+	)
+}
+
+class BoosterNotificationsService {
+	// MARK: - Init
+	init(store: Store, client: Client) {
+		self.store = store
+		self.client = client
+	}
+	
+	// MARK: - Protocol BoosterNotificationsServiceProviding
+	
+	func downloadBoosterNotifications(
+		completion: @escaping (Result<[Rule], HealthCertificateValidationError>) -> Void
+	) {
+		self.rulesDownloadService = RulesDownloadService(store: self.store, client: self.client)
```

Comment:
the `RulesDownloadService` should not be initialized in the download method. Otherwise we would init every time we trigger a download. 
Better would be to inject it in the init(). Additionally, it would make it easier for testing later because you can inject a mock and also the logic is decoupled.

Reply:
Yes this was just a temporary code, as the actual injection is handled in another task, but i will update it now